src/main/java/teammates/ui/controller/InstructorCourseDetailsPageData.java

```diff
@@ -20,11 +24,148 @@ public InstructorCourseDetailsPageData(AccountAttributes account) {
 
     public InstructorAttributes currentInstructor;
     public CourseDetailsBundle courseDetails;
-    public List<StudentAttributes> students;
     public List<InstructorAttributes> instructors;
     public String studentListHtmlTableAsString;
+    public ElementTag giveCommentButton;
+    public ElementTag courseRemindButton;
+    public CourseDetailsStudentsTable studentsTable;
     
-    public String getInstructorCourseRemindLink() {
+    public void init(InstructorAttributes currentInstructor, CourseDetailsBundle courseDetails, 
+                     List<InstructorAttributes> instructors, List<StudentAttributes> students) {
+        this.currentInstructor = currentInstructor;
+        this.courseDetails = courseDetails;
+        this.instructors = instructors;
+        
+        boolean isDisabled = !currentInstructor.isAllowedForPrivilege(
+                                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS);
+        
+        String content = "<span class=\"glyphicon glyphicon-comment glyphicon-primary\"></span>";
```

Comment:
Is there a more informative name than `content`?
I mean, I know it's the content of the ElementTag (as the attribute name suggests), but why is there HTML in the content?
What does it represent? Some sort of icon?
(this happens below too)